src/bot/discord_bot.py

```diff
@@ -423,18 +424,64 @@ async def _add_user(m, options):
     },
 }
 
-# BEGIN CLIENT EVENTS -----
+# SLASH COMMANDS
 
+tree = app_commands.CommandTree(client)
 
 @client.event
 async def on_ready() -> None:
     await on_startup()
+    await tree.sync(guild=discord.Object(id=1080421430426091551))
```

Comment:
I assume this is where the commands are added to the server/channel/guild.

I store server IDs in a db. We could create a tree_sync function that runs in on_ready. Looping through each server_id in a db query - or even all server id's that this bot can currently see (ie: has been invited to)